### ISSUE TITLE
/rftdim safedel <id> gives "Command is not allowed!" even when enabled

### ISSUE THREAD

**Issue by mctom987** [McJtyMods/RFTools#759]
Using a stock FTB IE 2.6.0 for MC 1.7.10
rftools-4.23.jar

I have modified config/rftools/main.cfg to enable players deleting owned dimensions, but I am still unable to delete my own dimensions. Console confirms dimension is owned by me.

```
/rftdim info 30
[03:06:31] [Server thread/INFO]: ?eDimension ID 30
[03:06:31] [Server thread/INFO]: ?eDescription string @tNoon,TFlat
[03:06:31] [Server thread/INFO]: ?eOwned by: mctom987
```

```
# If this is enabled then regular players can delete their own dimensions using the /rftdim safedel <id> command
B:playersCanDeleteDimensions=true
```

In fact, all /rft\* commands are giving the same "Command is not allowed!". Even the "help" commands. I can run them all from the console just fine, but I would like _players_ to be able to delete their own dimensions without enabling the "dangerous" editor method. I've perused the RFTools Wiki, but found nothing related to permissions other than that one-line change.

**Comment by McJty** [comment]
Hmm strange. But as an aside. Why is the editor method dangerous? It is not more dangerous then safedel.

Anyway, you do this command as the player that owns the dimension?

**Comment by mctom987** [comment]
Yeah, doing it as the owner.

Dangerous? Those are your words, not mine XD
https://github.com/McJty/RFTools/wiki/RFTools-Servers#nuking-dimensions

I haven't tested that yet. I'll give it a try though. I just prefer the command method.

**Comment by mctom987** [comment]
TNT method works, which I'm starting to like more. It gives (albeit a rather small) a cost to deleting dimensions, which can help curb abuse.

I did some additional testing, and I'm thinking the bug actually doesn't originate from RFTools. I opped myself with level 0, and that allowed me to delete only my own dimensions, as well as run "help". Without recompiling or running it through a debugger, I believe the default is permission level _NULL_, or something to the same effect. When you check permissions, you're doing a comparison to INT 0, of which NULL is not. I'm not sure if this is something RFTools _should_ handle, but it's definitely something that _could_ be handled.

While the server I intend to use this on is small enough I could probably justify opping every player at level 0, I'm satisfied with the TNT method instead. Depending on my time, I might find a way to fix this either at the config level, or a patch to Forge or RFTools.
